polaris-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -19,6 +19,7 @@
         outline: var(--p-border-width-2) solid
           var(--p-color-border-interactive-focus);
         outline-offset: var(--p-space-05);
+        border-radius: var(--p-border-radius-05);
```

Comment:
I feel like this isn't needed, outline should take the proper border-radius of checkbox

Reply:
im gonna revert this in [this PR](https://github.com/Shopify/polaris/pull/9436) as its the wrong focus style for checkbox